monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -296,6 +302,11 @@ class ReportPageComponent extends AuthComponent {
                       critical.</li> : null}
                   {this.state.report.overview.issues[this.Issue.MSSQL] ?
                     <li>MS-SQL servers are vulnerable to remote code execution via xp_cmdshell command.</li> : null}
+                  {this.state.report.overview.issues[this.Issue.DRUPAL] ?
+                    <li>Drupal servers are susceptible to a remote code execution vulnerability
+                      (<a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-6340">
+                      CVE-2019-6340</a>).</li> : null}
+                  {this.generateZeroLogonOverview()}
```

Comment:
```suggestion
                  {this.generateZerologonOverview()}
```
Let's change this ("l" instead of "L") everywhere for consistency